src/test/java/org/cactoos/func/CallableOfTest.java

```diff
@@ -54,19 +52,27 @@ public void convertsFuncIntoCallable() throws Exception {
     public void convertsRunnableIntoCallable() throws Exception {
         final AtomicBoolean flag = new AtomicBoolean(false);
         new CallableOf<>(
-            (Runnable) () -> { flag.set(true); }
+            () -> { flag.set(true); }
         ).call();
         MatcherAssert.assertThat(
             flag.get(),
             Matchers.is(true)
         );
     }
 
+    @SuppressWarnings("PMD.AvoidThrowingRawExceptionTypes")
+    @Test(expected = Exception.class)
```

Comment:
@llorllale Well, ``Exception`` here doesn't guarantee that it's the one from inside the Runnable, right? Maybe ``throw IllegalStateException ("from runnable")`` and here ``expected = IllegalStateException.class)`` -- this way you can also get rid of the PMD warning.

Reply:
@amihaiemil is the intention here that `CallableOf` wraps the runtime exception into a checked exception or not?

Reply:
@llorllale Ah, ``public T call() throws Exception`` -- very poor choice of design. When I see ``throws Exception`` I read it as "programmer had no idea how to handle exceptions"... I think this one is even from ``Callable``, so JDK. Nice; then leave the ``expected`` like that. But you can still throw ``IllegalStateException`` to get rid of the warning, I think, no?

Reply:
@amihaiemil ok, switching to `IllegalStateException`